src/olympia/files/tasks.py

```diff
@@ -40,9 +45,13 @@ def repack_fileupload(results, upload_pk):
                 extract_zip(upload.path, tempdir=tempdir)
 
                 if waffle.switch_is_active('enable-manifest-normalization'):
+                    xpi_data = parse_xpi(upload.path, minimal=True)
```

Comment:
That will cause us to parse the manifest twice, which is a bit unfortunate. We could fix this with a lot of refactoring (ideally `parse_xpi` would give us access to the manifest data so we don't have to call `ManifestJSONExtractor` later), but it doesn't block deploying this.